### DISCUSSION TITLE
Welcome to peppermint Discussions!

### DISCUSSION THREAD

**Discussion by potts99** [Peppermint-Lab/peppermint#76] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Cyberlytical-LLC** [comment]
@potts99 Is this still being developed? So far I really like what you have done and would love to use this for my I.T. company. I did have some ideas for it if you are still in development.

**Reply by potts99** [reply]
Yeah still in development, just struggling to find time at the moment, working under the next branch with an update coming at the end of august

**Comment by mattmadairy** [comment]
@potts99 Hey Jack, found your repository thru NetworkChuck's youtube channel. If possible, can you add functionality to manage preferences such as date/time format? Also, I'd like my clients to be able to submit their own tickets. In addition, id like to be able to disable and delete user accounts to keep my engineer list a little cleaner. Thanks for doing what you do!

**Reply by potts99** [reply]
@mattmadairy client portal now added 👍

**Reply by atb007** [reply]
Hey jack! clients can login to the portal as same internal users right?

**Comment by thaddeuscleo** [comment]
Hello Jack, i really like the idea of peppermint.sh as platform to provide ticketing service. I want to know is it possible to contribute to this project? If it's possible, is there any guidelines?

**Reply by potts99** [reply]
@thaddeuscleo writing these over the weekend if still interested

**Reply by thaddeuscleo** [reply]
Hey jack, sure im still interested
